index.html

```diff
@@ -79,24 +81,90 @@ <h1>Tonic</h1>
   <li><img src="./image/Ellipse 5.png" alt="ellipse"></li>
   <li>2015</li>
   </ul>
+</div>
   <p>A daily selection of privately presonalizaed reads: no accounts or singn-up required.</p>
   <ul class="skill">
-    <li><h4>html</h4></li>
-  <li>css</li>
-  <li>javaScript</li>
+    <li><button>html</button></li>
+  <li><button>css</button></li>
+  <li class="javaScript"><button>javaScript</button></li>
   </ul>
-  <button class="btn"><a href="#">See Project</a></button>
+  <a class="see_project" href="#"><button>See Project</button></a>
+</div>
+<div class="snapshoot">
+  <img src="./image/Snapshoot Portfolio (1).png" alt="snapchat">
+  <div class="snapshoot_1">
+  <h1>Multi_Post Stories</h1>
+  <ul class="profile">
+  <li>CANOPY</li>
+  <li><img src="./image/Ellipse 5.png" alt="ellipse"></li>
+  <li>Back End Dev</li>
+  <li><img src="./image/Ellipse 5.png" alt="ellipse"></li>
+  <li>2015</li>
+  </ul>
+</div>
+  <p>A daily selection of privately presonalizaed reads: no accounts or singn-up required.</p>
+  <ul class="skill">
+    <li><button>html</button></li>
+  <li><button>css</button></li>
```

Comment:
Please check that you don't have a fixed height on your project card. I can see that one of your card has it content overflowing